lib/src/controls/navigation/navigation_view/pane_items.dart

```diff
@@ -664,6 +663,13 @@ class __PaneItemExpanderState extends State<_PaneItemExpander>
     if (_open) {
       controller.value = 1;
     }
+
+    flyoutController.addListener(() {
```

Comment:
Be wary! This method will be called each time one of the inherited widgets you depend on in the build method has changed (and signals it has indeed) so you could end up adding listeners multiple times. Add some sort of check or move this call